### PR TITLE
frontend/sdk: Update available liquidity check

### PR DESCRIPTION
**Pull request by miguelmota**

### CODE REVIEW

File: packages/sdk/src/HopBridge.ts [hunk 1]
```diff
@@ -801,7 +774,47 @@ class HopBridge extends Base {
...
-    const availableLiquidity = credit.sub(debit)
+    let availableLiquidity = credit.sub(debit)
+
+    const isBondableChain = bondableChains.includes(sourceChain.slug)
+    const includePendingAmount =
+      isBondableChain && destinationChain.equals(Chain.Ethereum)
+
+    if (includePendingAmount) {
+      const bridgeContract = await this.getBridgeContract(sourceChain)
+      const pendingAmount = await bridgeContract.pendingAmountForChainId(
```
Comment (shanefontaine) [inline packages/sdk/src/HopBridge.ts:785]: IIUC this can be removed.

Comment (miguelmota) [reply packages/sdk/src/HopBridge.ts:785]: remove the pending amounts subtraction?

Comment (miguelmota) [reply packages/sdk/src/HopBridge.ts:785]: my understanding was having `((availableLiquidity - pendingAmounts) / 2) - 50k`

File: packages/sdk/src/HopBridge.ts [hunk 2]
```diff
@@ -457,39 +458,11 @@ class HopBridge extends Base {
...
-        .add(pendingAmounts)
-        .add(hTokenAmount) // account for transfer root bonds
-    }
-
     return {
       amountOut,
       rate,
       priceImpact,
-      requiredLiquidity,
+      requiredLiquidity: hTokenAmount,
```
Comment (shanefontaine) [inline packages/sdk/src/HopBridge.ts:465]: Will this affect [this line](https://github.com/hop-protocol/hop/blob/8eee9b4c3c594ea65a45489984f128e7d830ba9b/packages/frontend/src/pages/Send/Send.tsx#L431) downstream?

File: packages/sdk/src/HopBridge.ts (hunk 1 above)
Comment (shanefontaine) [reply packages/sdk/src/HopBridge.ts:785]: Maybe I'm not seeing it correctly, but it looks like this line 785 is not doing anything?

The equation you said is on line 812.

Comment (miguelmota) [reply packages/sdk/src/HopBridge.ts:785]: sorry i see the duplication now, will remove

File: packages/sdk/src/HopBridge.ts [hunk 3]
```diff
@@ -801,7 +774,47 @@ class HopBridge extends Base {
...
+        token.canonicalSymbol
+      )
+      const tokenPriceBn = parseUnits(tokenPrice.toString(), token.decimals)
+      const bufferAmountBn = parseUnits(PendingAmountBuffer, token.decimals)
+      const precision = parseUnits('1', token.decimals)
+      const bufferAmountTokensBn = bufferAmountBn
+        .div(tokenPriceBn)
+        .mul(precision)
+
+      availableLiquidity = availableLiquidity
```
[the remark this answers is not in the document]
Comment (miguelmota) [reply packages/sdk/src/HopBridge.ts:812]: you're correct, should be sub

File: packages/sdk/src/HopBridge.ts (hunk 2 above)
Comment (miguelmota) [reply packages/sdk/src/HopBridge.ts:465]: requiredLiquidity is simply the amount the bonder will need to bond (in this case it's the hToken amount output from swap)

File: packages/sdk/src/HopBridge.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (shanefontaine) [reply packages/sdk/src/HopBridge.ts:809]: @miguelmota I think it also needs the parenthesis here, assuming BigNumber follows PEMDAS. So adding `()` around `availableLiquidity.sub(pendingAmounts)`

File: packages/sdk/src/HopBridge.ts (hunk 2 above)
Comment (shanefontaine) [reply packages/sdk/src/HopBridge.ts:465]: Understood. I was reading this incorrectly. Makes sense & will resolve this now.

**Pull request merged**